frontend/src/features/projects/common/projectConverter.ts

```diff
@@ -119,15 +115,15 @@ const toApiProperty = (property: IProperty): IApiProperty => {
         parcelId: property.propertyTypeId === 0 ? property.id : undefined,
         buildingId: property.propertyTypeId === 1 ? property.id : undefined,
         value: property.assessed,
-        date: property.assessedDate ?? new Date(),
+        date: property.assessedDate ?? formatDate(new Date()),
```

Comment:
Is it always valid to default to today's date?

Reply:
This is my current assumption, that basically every time someone saves a project with a financial value they are essentially re-validating that value for the current moment in time.